src/main/java/core/basesyntax/exception/ValidationDataException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.exception;
+
+public class ValidationDataException extends RuntimeException {

```

Comment:
add an additional constructor that will get an additional parameter Throwlable to put all chain fo information from checked exception 
https://stackoverflow.com/questions/12568340/add-custom-message-to-thrown-exception-while-maintaining-stack-trace-in-java